docs/explanation/security/secure-deployment.rst

```diff
@@ -0,0 +1,67 @@
+====================================
+Best practices for secure deployment
+====================================
+
+Incorporate security from the initial setup of your MicroCeph instance.
+
+Network architecture
+--------------------
+
+* Segmentation: If the MicroCeph host has multiple network interfaces, configure Ceph's
+  public_network and cluster_network settings appropriately (check MicroCeph docs for details),
```

Comment:
Use inline formatting here too for public_network and cluster_network: https://canonical-starter-pack.readthedocs-hosted.com/latest/reference/rst-syntax-reference/#inline-formatting

"check MicroCeph docs for details" - could you please link to a specific section of the docs?